### ISSUE TITLE
A* search strategy integration

### ISSUE THREAD

**Issue by leventeBajczi** [ftsrg/theta#214]

**Comment by leventeBajczi** [comment]
If I understood correctly, this will be left out of this year's submission. I won't close this issue, please do try and iron out the bugs, and integrate the search strategy with a recent version of Theta!

**Comment by AdamZsofi** [comment]
@szdan97 is this planned?

**Comment by szdan97** [comment]
I don't think it will be integrated until this year's submission either. Asztrik is not working on this topic anymore, and I won't have time to do it in the near future. AFAIK there still were some bugs (incorrect results with A* for inputs which were correct with BFS) in the latest implementation, and only one version of the Hierarchical A* algorithm seemed to be efficient, which was in my opinion closer to some kind of lazy refinement than to the original idea of Hierarchical A*. Because of this, I'm not sure if working on merging all A* versions with their current implementation to the current master (which changed a lot since last year) would be worth it.
